ensure_ocm_cli.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Add a comment  briefly explaining what this script does and why it's needed.

Reply:
Added